Makefile

```diff
@@ -19,7 +19,7 @@ rebuild:
 # Fetch data from S3 and load to Postgres.
 # State should be specified as a lowercase, two-letter abbreviation, e.g. 'ca'.
 # Example usage: make load_representative_points state='ca'
-S3_BUCKET='https://s3-us-west-1.amazonaws.com/network-adequacy/data-011317/etl/output/'
+S3_BUCKET='https://s3-us-west-1.amazonaws.com/network-adequacy/data-02-15-18/etl/output/'
```

Comment:
Should we reverse naming? and have data inside etl/output/date ?

Reply:
I'm copying over the entire `data` directory and appending the date. But agreed we could have a better system.